packages/background/src/controllers/GasPricesController.ts

```diff
@@ -294,10 +298,10 @@ export class GasPricesController extends BaseController<GasPricesControllerState
                 );
 
                 // Get eth_feeHistory
-                // gets 10%, 25% and 50% percentile fee history of txs included in last 5 blocks
+                // gets 25%, 50% and 75% percentile fee history of txs included in last 10 blocks
                 const feeHistory: FeeHistory = await provider.send(
                     'eth_feeHistory',
-                    ['0x5', 'latest', [10, 25, 65]]
+                    ['0xA', 'latest', [25, 50, 75]]
```

Comment:
been testing it and I think using the 85th or 90th percentile for the **High** value would be better